python/plugins/ResourceSharing/test/data/collections/test_collection/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Does this license apply to the pokemon images too? I'm a bit concerned we might run into trouble if we distribute them